DESIGN.md

```diff
@@ -22,8 +22,8 @@ Kiwix uses themes to apply styles to views. This means that instead of adding a
 Themes are defined in res/values/themes.xml or res/values-night/themes.xml depending on night/day.
 The following steps should be followed to set a color or style of a view.
 1. Define the color or style:
-    a. Define the color in core/colors.xml. Use good naming (denim_blue, min_shaft_gray etc..).
-    b. Define the style in core/styles.xml. Follow the naming convention. TODO: (what is the naming convention for styles?)
+    * Define the color in core/colors.xml. Use good naming (denim_blue, min_shaft_gray etc..).
+    * Define the style in core/styles.xml. Follow the naming convention. TODO: (what is the naming convention for styles?)
```

Comment:
So there are 2 usages of a style
1. When we set a style to a theme attribute, this is how we define the default way a widget displays, here they are named to reflect that `Widget.KiwixTheme.[Button/TextView/etc]` and inherit from the correspondingly named parent, which can be kind of a crapshoot to find but hopefully is `Widget.MaterialComponents.CardView`
1. When there are views that are visually the same but distinct from our default styling then a style can be used so we keep our XML DRY. There aren't many great examples in the codebase tbh, `no_list_content_text` might be alright but it does define its positioning which is generally a no-no as a style (but it made the xml for those views a bit nicer) shouldn't define where you go just what you look like. `list_item_title` is a bit interesting too as this allows for some indirection but at the end of the day these should probably be theme attributes